react/components/ProductSummaryPrice.js

```diff
@@ -36,6 +36,11 @@ const ProductSummaryPrice = ({
     [`${productSummary.priceContainer} pv5`]: !showBorders,
   })
 
+  const sellingPrice = prop('Price')(commertialOffer)
```

Comment:
`const sellingPrice = prop('Price', commertialOffer)` to maintain conciseness with your other changes